lib/config/validate-config.js

```diff
@@ -45,6 +45,12 @@ function validateAbsolutePaths(value) {
   }
 }
 
+function validateBasePath(value) {
+  if (typeof value !== 'string' || value === '') {
+    return 'non-empty string';
+  }
+}
```

Comment:
Doesn't fusspot's `required` take care of checking for empty strings?

Reply:
I think fusspot's `required` takes care of `null` and `undefined`, an empty string is still a string so I don't think `required` would reject it.